ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/time-remaining/time-remaining.utils.js

```diff
@@ -0,0 +1,11 @@
+function getTimeBreakdown (milliseconds) {
```

Comment:
I can see this being useful elsewhere in the app, ex. in the transaction list where we'll show the time remaining for a pending transaction. I'll probably pull this out into the generic utils folder at that point.